doc/source/contributing.rst

```diff
@@ -349,18 +357,18 @@ Furthermore, it is recommended to have all `optional dependencies
 <http://pandas.pydata.org/pandas-docs/dev/install.html#optional-dependencies>`_
 installed. This is not strictly necessary, but be aware that you will see some error
 messages. Because all the code in the documentation is executed during the doc
-build, the examples using this optional dependencies will generate errors.
+build, the examples using the optional dependencies will generate errors.
 Run ``pd.show_versions()`` to get an overview of the installed version of all
 dependencies.
 
 .. warning::
 
-   Sphinx version >= 1.2.2 or the older 1.1.3 is required.
+   Sphinx version >= 1.2.2, or older than 1.1.3, is required.
```

Comment:
@jorisvandenbossche we should make an issue about just using newer sphinx no reason to support older any longer?